checkbox-support/checkbox_support/parsers/tests/test_v4l2_compliance.py

```diff
@@ -0,0 +1,64 @@
+import subprocess as sp
+from checkbox_support.parsers.v4l2_compliance import parse_v4l2_compliance
+import unittest as ut
+from unittest.mock import patch, MagicMock
+from pkg_resources import resource_filename
+
+
+def read_file_as_str(name: str):
+    resource = "parsers/tests/v4l2_compliance_data/{}.txt".format(name)
+    filename = resource_filename("checkbox_support", resource)
+    with open(filename) as f:
+        return f.read()
```

Comment:
Ah I see, I used pkg_resource because I saw another parser test using that to get the test input files. (Also does this mean we are moving towards python 3.9? :D)